samples/excel/range/copy-move-multiply-values.yaml

```diff
@@ -0,0 +1,276 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: 'Copy, move and multiply values'
+description: 'Explore the APIs to copy, move and multiply values in a range'
+script:
+  content: |
+    $("#setup").click(setup);
+
+    $("#copy-values").click(copyValues);
+    $("#move-values").click(moveValues);
+    $("#multiply-values-for-loop").click(multiplyValuesForLoop);
+    $("#multiply-values-map").click(multiplyValuesMap);
+
+    async function setup() {
+        try {
+            await Excel.run(async (context) => {
+
+                await resetWorksheet(context.workbook.worksheets.getActiveWorksheet());
+                
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let data = [
+                    ["Product", "Qty", "Unit Price", "Total Price"],
+                    ["Almonds", 2, 7.50, "=C3 * D3"],
+                    ["Coffee", 1, 34.50, "=C4 * D4"],
+                    ["Chocolate", 5, 9.56, "=C5 * D5"]
+                ];
+
+                let range = sheet.getRange("B2:E5");
+                range.values = data;
+                range.format.autofitColumns();
+
+                let header = sheet.getRange("B2:E2");
+                header.format.fill.color = "#4472C4";;
+                header.format.font.color = "white";
+
+                await context.sync();
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    async function resetWorksheet(sheet: Excel.Worksheet) {
+        let context = <Excel.RequestContext>sheet.context;
+
+        sheet.load("name, position");
+
+        await context.sync();
+
+        let newSheet = context.workbook.worksheets.add();
+        newSheet.position = sheet.position;
+        sheet.delete();
+        newSheet.name = sheet.name;
+        newSheet.activate();
+
+        await context.sync();
+    }
+
+    async function copyValues() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let fromRange = sheet.getRange("B2:E5");
+                fromRange.load("values");
+
+                await context.sync();
+                
+                let toRange = sheet.getRange("B10:E13");
```

Comment:
Likely, but it also introduces the more advanced range functions like getResizedRange earlier. I prefer to keep things simple and ramp up gradually.